blueocean-dashboard/src/main/js/components/LogConsole.jsx

```diff
@@ -1,7 +1,5 @@
 import React, { Component, PropTypes } from 'react';
-import { scrollToHash } from './ScrollToHash';
-
-const { array, bool, string } = PropTypes;
+import { scrollHelper } from './ScrollHelper';
 
 const INITIAL_RENDER_CHUNK_SIZE = 100;
 const INITIAL_RENDER_DELAY = 300;
```

Comment:
Where does this number come from? A 300ms pause in the UI will be pretty seriously noticeable to users. And will it cease to be enough if more components are instantiated, or the user happens to have YouTube running in another tab? Will it be enough if the user isn't looking at this tab and it has been de-prioritised? Surely there's something deterministic we can hook into from React?